NuSpecs/Microsoft.WinRTC.libwebrtc.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Microsoft.WinRTC.libwebrtc</id>
+        <version>84.0.3-alpha</version>
+        <authors>Microsoft</authors>
+        <owners>Microsoft</owners>
+        <projectUrl>http://github.com/microsoft/winrtc</projectUrl>
+        <license type="file">license.txt</license>
+        <requireLicenseAcceptance>true</requireLicenseAcceptance>
+        <releaseNotes>First drop</releaseNotes>
```

Comment:
Please replace with: Upgrading WinRTC with the WebRTC changes up to version m84. Bringing the Media Foundation based H264 encoder and decoder from the WebRTC-UWP project.